src/java.base/share/classes/java/util/Locale.java

```diff
@@ -70,131 +70,150 @@
+ *   <dd> <em>BCP 47 deviation:</em> BCP 47 only
+ *   uses hyphen ('-') as a delimiter, {@code Locale} is more lenient.</dd>
+ *
+ *   <dd> <em>Example:</em> "polyton" (Polytonic Greek), "POSIX"</dd>
  *
  *   <dt><a id="def_extensions"><b>extensions</b></a></dt>
  *
- *   <dd>A map from single character keys to string values, indicating
- *   extensions apart from language identification.  The extensions in
- *   {@code Locale} implement the semantics and syntax of BCP 47
- *   extension subtags and private use subtags. The extensions are
+ *   <dd> A map from single character keys to string values, indicating
+ *   extensions apart from language identification.</dd>
+ *   <dd> <em> BCP 47 deviation:</em> The {@code
+ *   extensions} in {@code Locale} implement the semantics and syntax of BCP 47
+ *   extension subtags <em>and</em> private use subtags. The {@code extensions}
+ *   field cannot have empty values. </dd>
+ *
+ *   <dd> <em>Case convention:</em> {@code extensions} are
  *   case insensitive, but {@code Locale} canonicalizes all
- *   extension keys and values to lower case. Note that extensions
- *   cannot have empty values.</dd>
+ *   extension keys and values to lower case.</dd>
  *
- *   <dd>Well-formed keys are single characters from the set
+ *   <dd> <em>Syntax:</em> Well-formed keys are single characters from the set
  *   {@code [0-9a-zA-Z]}.  Well-formed values have the form
  *   {@code SUBTAG ('-' SUBTAG)*} where for the key 'x'
- *   <code>SUBTAG = [0-9a-zA-Z]{1,8}</code> and for other keys
- *   <code>SUBTAG = [0-9a-zA-Z]{2,8}</code> (that is, 'x' allows
+ *   {@code SUBTAG = [0-9a-zA-Z]{1,8}} and for other keys
+ *   {@code SUBTAG = [0-9a-zA-Z]{2,8}} (that is, 'x' allows
  *   single-character subtags).</dd>
  *
- *   <dd>Example: key="u"/value="ca-japanese" (Japanese Calendar),
+ *   <dd> <em>Example:</em> key="u"/value="ca-japanese" (Japanese Calendar),
  *   key="x"/value="java-1-7"</dd>
  * </dl>
  *
- * <b>Note:</b> Although BCP 47 requires field values to be registered
+ * <b>BCP 47 deviation:</b> Although BCP 47 requires field values to be registered
  * in the IANA Language Subtag Registry, the {@code Locale} class
- * does not provide any validation features.  The {@code Builder}
+ * does not validate this requirement. For example, the variant code <em>"foobar"</em>
+ * is well-formed since it is composed of 5 to 8 alphanumerics, but is not defined
+ * the IANA Language Subtag Registry. The {@link Builder}
  * only checks if an individual field satisfies the syntactic
  * requirement (is well-formed), but does not validate the value
- * itself.  See {@link Builder} for details.
+ * itself. Conversely, {@link #of(String, String, String) Locale::of} and its
+ * overloads do not make any syntactic checks on the input.
  *
- * <h2><a id="def_locale_extension">Unicode locale/language extension</a></h2>
+ * <h3><a id="def_locale_extension">Unicode locale/language extension</a></h3>
```

Comment:
Probably using the exact name in the external spec is better [Unicode BCP 47 U Extension](https://unicode.org/reports/tr35/#u_Extension)